### PR TITLE
Add opentelemetry.io docs

### PR DESCRIPTION
**Pull request by austinlparker**
Per open-telemetry/opentelemetry.io#472, we're mirroring the docs content on the website to each SIG. When a release occurs and these docs are updated, please make an issue or PR mirroring them to their appropriate location in the website repo (https://github.com/open-telemetry/opentelemetry.io/tree/main/content/en/docs/collector).

### CODE REVIEW

File: website_docs/configuration.md
Comment (jpkrohling) [inline website_docs/configuration.md:46]: It's probably about time to start using the new port everywhere, 4317 instead of 55680 :-)

Comment (shelbyspees) [reply website_docs/configuration.md:46]:
```suggestion
    endpoint: otelcol:4317
```

**Comment by tigrannajaryan** [comment]
@austinlparker please address the comment.

File: website_docs/configuration.md
Comment (shelbyspees) [inline website_docs/configuration.md:82]:
```suggestion
        endpoint: 0.0.0.0:4317
```

File: website_docs/configuration.md
Comment (shelbyspees) [inline website_docs/configuration.md:90]:
```suggestion
    endpoint: otelcol:4317
```

File: website_docs/configuration.md
Comment (shelbyspees) [inline website_docs/configuration.md:92]:
```suggestion
    endpoint: otelcol2:4317
```

File: website_docs/configuration.md
Comment (shelbyspees) [inline website_docs/configuration.md:321]:
```suggestion
    endpoint: "otelcol2:4317"
```

File: website_docs/configuration.md
Comment (shelbyspees) [inline website_docs/configuration.md:325]:
```suggestion
    endpoint: otelcol2:4317
```

File: website_docs/configuration.md
Comment (shelbyspees) [inline website_docs/configuration.md:330]:
```suggestion
    endpoint: https://example.com:4317/v1/traces
```

File: website_docs/getting-started.md
Comment (shelbyspees) [inline website_docs/getting-started.md:78]: If I understand the spec correctly, it's only the gRPC port that needs to be updated. Please correct me if I'm wrong here.

```suggestion
      -p 55678-55679:55678-55679 -p 4317:4317 -p 8888:8888 -p 9411:9411 \
```

File: website_docs/configuration.md
Comment (shelbyspees) [reply website_docs/configuration.md:46]: Sorry, noob mistake. I think I got them all this time. @austinlparker feel free to merge in the suggestions if they look right.

**Comment by austinlparker** [comment]
The link check is never gonna work on this, is there a way to ignore it for files in this path?

**Comment by shelbyspees** [comment]
@austinlparker it doesn't look like markdown-link-check supports ignoring files, just [links on specific lines or in specific blocks of markdown](https://github.com/gaurav-nelson/github-action-markdown-link-check#disable-check-for-some-links).

> `<!-- markdown-link-check-disable -->` and `<!-- markdown-link-check-enable-->`: Use these to disable links for all links appearing between these comments.

I don't love that option since it adds noise to the source files. It doesn't look like we currently use markdown-link-check in the main docs repo, but if we decide to in the future we'd have to deal with any disable/enable comments we add in here.

There's also an open issue to add an `exclude` parameter to the GitHub action config: gaurav-nelson/github-action-markdown-link-check#38. It looks like the maintainer wants to add it, but I'm not sure about their timeline for that.

**Comment by tigrannajaryan** [comment]
@austinlparker Is the docs content in this PR copied from https://github.com/open-telemetry/opentelemetry.io/tree/main/content/en/docs/collector ?

Do you also plan to add the update-docs job in a later PR like we have for Java https://github.com/open-telemetry/opentelemetry-java/blob/main/.github/workflows/docs-update.yml ?

**Comment by austinlparker** [comment]
> @austinlparker Is the docs content in this PR copied from https://github.com/open-telemetry/opentelemetry.io/tree/main/content/en/docs/collector ?

Yeah, it's whatever was there when I made the PR.

> Do you also plan to add the update-docs job in a later PR like we have for Java https://github.com/open-telemetry/opentelemetry-java/blob/main/.github/workflows/docs-update.yml ?

Yes, if it's desired.

**Comment by austinlparker** [comment]
I went ahead and pushed the update script in this PR as well, can be removed if desired.

**Review by alolita — APPROVED** [review]

**Review by carlosalberto — APPROVED** [review]

**Comment by tigrannajaryan** [comment]
> I went ahead and pushed the update script in this PR as well, can be removed if desired.

Thanks.

What's the plan with the failing link checks? Will you add `<!-- markdown-link-check-disable -->` annotations or you are looking for some other solution?

**Comment by austinlparker** [comment]
I'll add the annotations for now, I was hoping someone had a better idea on ignoring the path but this should at least unblock things for now.

**Comment by austinlparker** [comment]
I guess bounce the checks when the GitHub Actions incident is resolved?

File: .github/workflows/docs-update.yml
Comment (tigrannajaryan) [inline .github/workflows/docs-update.yml:15]: I assume once this PR is merged we will need to add the token, right? Let's do it after it is merged and do a test run of the workflow to see how it works.

**Review by tigrannajaryan — APPROVED** [review]

File: .github/workflows/docs-update.yml
Comment (tigrannajaryan) [reply .github/workflows/docs-update.yml:15]: @austinlparker can you please add the token and let's see how the docs-update works.

**Pull request merged**
